activation/atxdb.go

```diff
@@ -617,6 +630,10 @@ func (db *DB) GetNodeLastAtxID(nodeID types.NodeID) (types.ATXID, error) {
 	if exists := nodeAtxsIterator.Last(); !exists {
 		return *types.EmptyATXID, ErrAtxNotFound(fmt.Errorf("atx for node %v does not exist", nodeID.ShortString()))
 	}
+	nodeAtxsIterator.Release()
```

Comment:
As commented elsewhere, move this into a `defer` statement immediately after getting the iterator